src/Streamly/Internal/Data/Stream/IsStream/Top.hs

```diff
@@ -387,7 +412,7 @@ mergeLeftJoin _eq _s1 _s2 = undefined
 --
 -- Time: O(m x n)
 --
--- /Unimplemented/
+-- /Pre-release/
```

Comment:
This function has a XXX comment below ("Need to use mutable array"), it does not work correctly, so let's mark it unimplemented unless we fix that.